### PR TITLE
[Rule Tuning] Exclude Windows Error Reporting & Printer Driver

### PR DESCRIPTION
**Pull request by nicpenning**
## Summary

These false positives have been found and make sense to exclude for any organization since they are Windows system files.

I kept there normal cases. I am not sure a lowercase is necessary or not for this type of query.

### CODE REVIEW

File: rules/windows/defense_evasion_code_injection_conhost.toml [hunk 1]
```diff
@@ -23,7 +23,8 @@ type = "query"
 
 query = '''
 event.category:process and event.type:(start or process_started) and
-  process.parent.name:conhost.exe
+  process.parent.name:conhost.exe and not
+  process.executable:("C:\Windows\splwow64.exe" or "C:\Windows\System32\WerFault.exe")
```
Comment (brokensound77) [inline rules/windows/defense_evasion_code_injection_conhost.toml:27]: One small nit, as leading with `not` helps with readability and is more consistent with existing rules

```suggestion
  process.parent.name:conhost.exe and 
  not process.executable:("C:\Windows\splwow64.exe" or "C:\Windows\System32\WerFault.exe")
```

**Review by brokensound77 — COMMENTED** [review]
Thanks for the contribution - left one small nit and can you bump the updated date

**Comment by nicpenning** [comment]
Good to know, thanks for reviewing!

**Review by Samirbous — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
LGTM 👍

**Pull request merged**